packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/TickLabels.test.tsx

```diff
@@ -50,4 +50,11 @@ describe('<TickLabels>', () => {
       expect(tick).toHaveStyle(`left: ${pos};`);
     });
   });
+  it('shows the correct value above each tick', () => {
+    expect(screen.getAllByText(/0/)).toBeTruthy();
```

Comment:
again, think we should avoid using `getAllByText`. use a selector that finds the exact element needed.

Reply:
I've made this test more strict now. See commit https://github.com/grafana/grafana/pull/61009/commits/8d0e8a83d7e7c7f3ccd78330bb7b11265e48f554.